modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json

```diff
@@ -5806,7 +5806,7 @@
                       "status": "404",
                       "detail": "Resource not found",
                       "source": {
-                        "pointer": "/Users/rockwellrice/Documents/oddball/vets-api/modules/claims_api/app/controllers/claims_api/v2/veterans/disability_compensation_controller.rb:74:in `attachments'"
+                        "pointer": "/Users/steven.cumming/workspace/va.gov/vets-api/modules/claims_api/app/controllers/claims_api/v2/veterans/disability_compensation_controller.rb:74:in `attachments'"
                       }
```

Comment:
There was a recent oversight (by me) on an update to our error reporting for this error which did not properly shorten this path here.  The only fix ATM is to manually shorten the path, `/modules/claims_api/app/controllers/claims_api/v2/veterans/disability_compensation_controller.rb:74:in `attachments'"` would probably be just right.  There is one other place this occurs which is on line 5809.